classes/Http/Controller/ProfileController.php

```diff
@@ -92,37 +100,31 @@ public function processAction(Request $request)
             $formData['speaker_photo'] = $request->files->get('speaker_photo');
         }
 
-        /** @var \HTMLPurifier $htmlPurifier */
-        $htmlPurifier = $this->service('purifier');
-
-        $form    = new SignupForm($formData, $htmlPurifier);
+        $form    = new SignupForm($formData, $this->purifier);
         $isValid = $form->validateAll('update');
 
         if ($isValid) {
             $sanitizedData = $this->transformSanitizedData($form->getCleanData());
 
             if (isset($formData['speaker_photo'])) {
-                /** @var ProfileImageProcessor $profileImageProcessor */
-                $profileImageProcessor = $this->service('profile_image_processor');
-
-                $sanitizedData['photo_path'] = $profileImageProcessor->process($formData['speaker_photo']);
+                $sanitizedData['photo_path'] = $this->profileImageProcessor
+                    ->process($formData['speaker_photo']);
```

Comment:
Not sure about the wrapping here, but I don't mind.

Reply:
Sorry, I guess my IDE did that automatically.